terraform/parameter_store.tf

```diff
@@ -0,0 +1,27 @@
+resource "aws_ssm_parameter" "wordpress-db-host" {
+  name        = "/wordpress/WORDPRESS_DB_HOST"
+  description = "The host parameter to be used by the container and DB"
```

Comment:
This value will only be used by the container

Reply:
I see, it will be used by the container to connect with the Database.